.buildkite/pipeline.yml

```diff
@@ -106,7 +106,7 @@ steps:
     # Note that 'refill-balance' below is the initial 'TASTY_CLEVELAND_MONEYBAG_SECRET_KEY' balance
     # which may need to be adjusted in case of insufficient balance errors
     - $(nix-build ci.nix -A run-chain-tests
-        --argstr refill-balance 200
+        --argstr refill-balance 5000
```

Comment:
:money_with_wings: just a reminder: if this is enough please remember to re-set it to a value close-ish to what was actually used.